### DISCUSSION TITLE
Real Time Tracking (Quests, Achievements, Crates' Progress) - Maybe Pinnable to the GUI

### DISCUSSION THREAD

**Discussion by kispintyi** [TheMysterys/MCCI-Utils#23] [Ideas]
It would be awesome if we can PIN the Quests, Achievements, Crates' Progress to the GUI or have a system similar like that when you get progress in something it would alert you in a toast, that you got progress for this and this, and if Pinnable to GUI is possible: There we can pin all the things we need to the GUI, and have no trouble with logging out of the game to see these (because in-game we have no permission to see these, if we know the commands that is)

**Comment by TheMysterys** [comment]
I'm going to hold off adding/implementing this cause I want to see what Noxcrew does with this (Likely a feature they will add in a future update) but I will also draft up a way for the mod to also create it's own way of doing this too
